tests/math/sequence/test_sequential_funcs.py

```diff
@@ -357,7 +437,6 @@ def test_prod_bkwd():
 )
 def test_multi_zero_prod_bkwd():
     """Drives tests cases with various configurations of zeros"""
-    pass
```

Comment:
Not sure how I feel about this but `tox` prefers it this way.

Reply:
`tox`? Do you mean the linter that gets run when you run the `format` job?

Reply:
I was referring to `tox -e format` from https://github.com/rsokl/MyGrad/blob/master/CONTRIBUTING.md.